src/selectors.js

```diff
@@ -1,9 +1,15 @@
 import { matchPath } from "react-router"
+import isObject from 'lodash/isObject'
 
 const createSelectors = (structure) => {
-  const { getIn, toJS } = structure
-  const getLocation = state => toJS(getIn(state, ['router', 'location']))
-  const getAction = state => toJS(getIn(state, ['router', 'action']))
+  const { get, toJS } = structure
+  const getRouter = state => {
+    const router = toJS(get(state, 'router'))
```

Comment:
Can we just use `getIn(state, 'router')` here? So, we don't need to create addition `get`.